src/components/question-genre/question-genre.jsx

```diff
@@ -0,0 +1,72 @@
+import React, {useState} from 'react';
+import {shape, string, arrayOf, func} from 'prop-types';
+import AnswerGenre from '../answer-genre/answer-genre.jsx';
+
+const QuestionGenreScreen = ({question, onAnswer}) => {
+  const {answers} = question;
+  const initialUserAnswers = answers.reduce((acc, _item, i) => ({...acc, [i + 1]: false}), {});
+
+  const [userAnswers, setUserAnswers] = useState(initialUserAnswers);
```

Comment:
А как ты считаешь, можно ли просто сделать так:

```suggestion
  const [userAnswers, setUserAnswers] = useState({});
```